captum/attr/_core/feature_ablation.py

```diff
@@ -601,3 +593,20 @@ def _find_output_mode(
             feature_mask is None
             or all(len(sm.shape) == 0 or sm.shape[0] == 1 for sm in feature_mask)
         )
+
+    def _run_forward(self, *args, **kwargs) -> Tensor:
+        forward_output = _run_forward(*args, **kwargs)
```

Comment:
nit: It seems a bit confusing when seeing both the instance method and original method named as _run_forward, could consider renaming this one slightly, but either way is fine.